python/ray/train/_internal/storage.py

```diff
@@ -302,6 +307,34 @@ def get_fs_and_path(
     return pyarrow.fs.FileSystem.from_uri(storage_path)
 
 
+def _read_file_as_str(
+    storage_filesystem: pyarrow.fs.FileSystem,
+    storage_path: str,
+    compression: Optional[str] = "detect",
+    buffer_size: Optional[int] = None,
+    encoding: Optional[str] = "utf-8",
+) -> str:
```

Comment:
I think I'm okay with adding this helper, but also it might be clearer to just do the filesystem operations directly? We can add this in the future if it turns out we need to read a file as text directly from cloud very often.